client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -253,10 +253,14 @@ class TransferDomainStep extends React.Component {
 	transferIsRestricted = () => {
 		const { submittingAvailability, submittingWhois } = this.state;
 		const submitting = submittingAvailability || submittingWhois;
+		const transferRestrictionStatus = get(
+			this.state,
+			'inboundTransferStatus.transferRestrictionStatus',
+			false
+		);
 
 		return (
-			! submitting &&
-			'not_restricted' !== this.state.inboundTransferStatus.transferRestrictionStatus
+			! submitting && transferRestrictionStatus && 'not_restricted' !== transferRestrictionStatus
```

Comment:
Since we're using `get` above now, do we still need to check `&& transferRestrictionStatus`?

Reply:
If we remove that, `'not_restricted' !== transferRestrictionStatus` is always true?